server/services/team/service.ts

```diff
@@ -509,7 +509,7 @@ export class TeamService extends ServiceClass implements ITeamService {
 		return this.TeamMembersModel.findByTeamIds(teamIds, options).toArray();
 	}
 
-	async members(uid: string, teamId: string, canSeeAll: boolean, { offset, count }: IPaginationOptions = { offset: 0, count: 50 }, { query }: IQueryOptions<ITeam>): Promise<IRecordsWithTotal<ITeamMemberInfo>> {
+	async members(uid: string, teamId: string, canSeeAll: boolean, { offset, count }: IPaginationOptions = { offset: 0, count: 50 }, query: Partial<FindOneOptions<IUser>>): Promise<IRecordsWithTotal<ITeamMemberInfo>> {
```

Comment:
I don't know if naming this `query` is misleading 🤔 since it's not an actual query, but options 😬

Reply:
Actually, the `FindOneOptions` type is wrong here -- it should be a `FilterQuery`. Tks 😄